app/src/main/java/org/oppia/app/story/StoryFragmentInterface.kt

```diff
@@ -0,0 +1,20 @@
+package org.oppia.app.story
+
+import android.content.Context
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+
+interface StoryFragmentInterface {
```

Comment:
Similarly, prefer removing 'interface' and instead naming this based on the functionality it actually provides. Are onAttach & onCreateView needed here anymore? If not, please remove.

That, in turn, just leaves scrolling. Perhaps 'StoryFragmentScroller' or similar would be clearer?

Reply:
Sounds good!